docs/source/whatsnew/0.8.4.txt

```diff
@@ -19,6 +19,8 @@ Enhancements
   passed to :class:`~zipline.algorithm.TradingAlgorithm` by the keyword argument
   ``create_event_context`` (:issue:`828`).
 
+* Added `isnan`, `notnan`, and `isfinite` methods to `Factor` (:issue:`861`).
```

Comment:
This is because they should not be pulled in by sphinx, they are just `.. include::`ed in the releases.rst page.